pop/POPAnimation.mm

```diff
@@ -270,3 +270,43 @@ - (id)pop_animationForKey:(NSString *)key
 }
 
 @end
+
+@implementation POPAnimation (NSCopying)
+
+- (instancetype)copyWithZone:(NSZone *)zone
+{
+  POPAnimation *copy = nil;
+  
+  // Must use [self class] instead of POPAnimation so that subclasses can call this via super.
+  // Since POPAnimation and POPPropertyAnimation are (semi-)concrete, wrap initialization in a try-catch.
+  // If regular initialization fails, use _init.
```

Comment:
My idea here was since -init throws an exception on POPAnimation and POPPropertyAnimation.

But the subclasses don't, so here I can try the normal -init, and if that throws an exception, just use -_init